catalog/admin/module/catalog-admin-module-sources/src/main/webapp/templates/deleteSource.handlebars

```diff
@@ -11,7 +11,10 @@
  *
  **/
  --}}
-<div class="delete-source-name">{{name}}</div>
+<div class="delete-source-name"><span style="margin-right: 2em">{{name}}</span>
```

Comment:
New iteration: 
![screen shot 2015-11-11 at 9 41 55 am](https://cloud.githubusercontent.com/assets/14062223/11096752/e41fcf00-8858-11e5-98cc-d0873cd5e04b.png)

The source checkbox (next to the source name) will select/de-select all services underneath. If one service is deselected, it will also deselect the source checkbox. If all services are manually checked, it will also check the source checkbox.